### ISSUE TITLE
training issue about PSNR

### ISSUE THREAD

**Issue by ss00atbupt** [tarun005/FLAVR#47]
Hi, tarun ,excellent work on video interpolation!
 I tried run your code , but I have some trouble. I set my config as
 `batch_size=2, beta1=0.9, beta2=0.99, checkpoint_dir='.', cuda=True, data_root='/home/Changchen/dataset./vimeo_septuplet', dataset='vimeo90K', exp_name='exp', joinType='concat', load_from=None, log_iter=60, loss='1*L1', lr=0.0002, max_epoch=50, model='unet_18', n_outputs=1, nbr_frame=4, nbr_width=1, num_gpu=1, num_workers=16, pretrained=None, random_seed=12345, resume=False, resume_exp=None, start_epoch=0, test_batch_size=1, upmode='transpose', use_tensorboard=False, val_freq=1`
At the beginning, psnr was normal about 20,but it has gradually decreased to about 14. I wonder why it seems to be misconvergence.
Thank you for any help!

**Comment by ss00atbupt** [comment]
By the way,I train my modle on vimeo-90k.

**Comment by ss00atbupt** [comment]
Hi,@tarun005 !
I tried to set lr as 0.002,now it seems better than before.My model's psnr has raised a little!
Should I set lr bigger than 0.002 at the beginning?
Thank you for any possible help!

**Comment by tarun005** [comment]
Can you make sure that the versions of the PyTorch are the same as recommended?
I did not face any such issues at my end.

**Comment by ss00atbupt** [comment]
> Thanks for your reply!
I didn't use the same PyTorch version .Somehow I can't use it , CUDA version can't match when I use PyTorch 10.1. So I used 10.2 . 
Does the version affects the training ?
